shared/constants/transaction.ts

```diff
@@ -412,7 +412,7 @@ export interface TransactionMeta {
    */
   submittedTime?: number;
   /** The error encountered during the transaction */
-  txErr?: TxError;
```

Comment:
Have we checked this is outdated and there is no use of `txErr` anywhere?

Reply:
Yes, I checked it only usage in this file. :+1:  I found it weird but as it's js everything else was working normally.